Documentation/ApiOverview/TypoScriptSyntax/Syntax/Conditions.rst

```diff
@@ -274,12 +263,106 @@ from the top level. This is normally done when TypoScript code from
 various records is combined.
 
 
+.. index:: TypoScript; Symfony expression language
 .. _typoscript-syntax-conditions-expression-language:
 
-Custom Conditions With Symfony Expression Language
-==================================================
+Custom conditions with the `Symfony`:pn: expression language
+============================================================
+
+Further information about how to extend TypoScript with your own custom
+conditions can be found within :ref:`sel-within-typoscript-conditions`.
+
+
+.. _typoscript-syntax-conditions-examples:
+
+Syntax examples
+===============
+
+Variables and functions
+-----------------------
+
+For a detailed list of the available objects and functions refer to the
+:ref:`the TypoScript Reference <t3tsref:conditions>`.
+
+Variables::
+
+   [page["backend_layout"] == 1]
+      page.42.value = Backend layout 1 choosen
+   [END]
+
+Functions::
+
+   [loginUser('*')]
+      page.42.value = Frontend user logged in
+   [END]
+   [getTSFE().isBackendUserLoggedIn()]
+      page.42.value = Backend user logged in
+   [END]
+
+Literals
+--------
+
+For a complete list have a look at the
+`SEL supported literals <https://symfony.com/doc/current/components/expression_language/syntax.html#supported-literals>`__.
+
+Strings::
+
+   [request.getNormalizedParams().getHttpHost() == 'www.example.org']
+      page.42.value = Http Host is www.example.org
+   [END]
+
+Arrays::
+
+   [page["pid"] in [17,24]]
+      page.42.value = This page is a child of page 17 or page 24
+   [END]
+
+
+Operators
+---------
+
+Please see a complete list of available operators here:
+`SEL syntax operators <https://symfony.com/doc/current/components/expression_language/syntax.html#comparison-operators>`__
+
+Equality::
+
+   [applicationContext == "Development"]
+      page.42.value = The application context is exactly "Development"
+   [END]
+
+Wildcards::
```

Comment:
Since `like()` is custom we should also mention `?` to match any single character.

Reply:
I added some information and a link to the definition in the TypoScript reference